iOS/Multiplatform Redux Sample/Multiplatform Redux Sample/Utils/RoundDisposalImage.swift

```diff
@@ -7,26 +7,26 @@
 
 import UIKit
 
-class RoundDisposalImage: UIView {
-    private let backgroundView = UIImageView.autoLayout()
+class RoundColoredImage: UIView {
+    private let backgroundView = UIView.autoLayout()
     private let imageView = UIImageView.autoLayout()
 
-    init(withSize: CGFloat, iconSize: CGFloat) {
+    init(withSize: CGFloat, iconSize: CGFloat, backgroundColor: UIColor) {
         super.init(frame: .zero)
         translatesAutoresizingMaskIntoConstraints = false
```

Comment:
I don't think this is necessary for the view to work. If that is correct I would prefer to use `.autoLayout()` in the containing view to make it consistent with other UIKit elements and to make it easier to spot where `.autoLayout()` is missing in case of layout errors.